h2o/src/main/scala/org/apache/mahout/h2obindings/H2OEngine.scala

```diff
@@ -32,6 +32,7 @@ import org.apache.mahout.common.{Hadoop1HDFSUtil, HDFSUtil}
 object H2OEngine extends DistributedEngine {
   // By default, use Hadoop 1 utils
   var hdfsUtils: HDFSUtil = Hadoop1HDFSUtil
+  val operations: DistributedOperations = null
```

Comment:
I guess in case of H20 this will break with NPE... Perhaps we can make it an Option[...] and gracefully report capability is not supported by engine if it happens to be None.